pkg/gitops/prepare/prepare_test.go

```diff
@@ -0,0 +1,139 @@
+/*
+Copyright 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package prepare
```

Comment:
I'd name it `gitops_prepare` or `gitopsprepare`

Reply:
I renamed the imports, since renaming the package itself would also imply the change of the folder name to `gitops/gitopsprepare`, which sounds redundant.